src/site/affordable-housing.liquid

```diff
@@ -0,0 +1,353 @@
+      <label for="{{opt}}">{{opt}}</label><br/>
+    {% endfor %}
+    </div>
+  </fieldset>
+
+  <fieldset>
+    <legend>Rent</legend>
+    <div>
+      <label for="rent-max">Maximum ($ per month)</label><br />
+      <input type="number" oninput="ToggleRentOption();" min="0" id="rent-max" name="rentMax" value="{{query.rentMax | escape}}"><br/>
+      <span id="include-unknown-rent-container" hidden>
+        <input type="checkbox" id="include-unknown-rent" name="includeUnknownRent"
+        {% if num_filters_applied == 0 or query.includeUnknownRent == "on" %}checked{% endif %}>
+        <label for="include-unknown-rent">Include properties with no rent information</label> 
+      </span>
+    </div>
+  </fieldset>
+
+  <fieldset>
+    <legend>Income</legend>
+    <div>
+      <label for="income">Yearly Income ($)</label>
+      <span class="tooltip_entry">
+        <span class="icon_query"></span>
+        <span class="tooltip_content">
+          Provide your income to hide properties you are not eligible for. Your information will not be stored.
+        </span>
+      </span>
+      <br />
+      <input type="number" oninput="ToggleIncomeOption();" min="0" id="income" name="income" value="{{ query.income | escape}}"><br/>
+      <span id="include-unknown-income-container" hidden>
+        <input type="checkbox" id="include-unknown-income" name="includeUnknownIncome"
+        {% if num_filters_applied == 0 or query.includeUnknownIncome == "on" %}checked{% endif %}>
+        <label for="include-unknown-income">Include properties with no income limit information</label>
+      </span>
+    </div>
+  </fieldset>
+
+  <fieldset>
+    <legend>More Filters</legend>
+    <div>
+      <input type="checkbox" id="wheelchair-accessible-only" name="wheelchairAccessibleOnly"
+        {% if query.wheelchairAccessibleOnly == "on" %}checked{% endif %}>
+      <label for="wheelchair-accessible-only">Only show wheelchair-accessible properties</label>
+    </div>
+    <div>
+      <input type="checkbox" id="include-referrals" name="includeReferrals"
+        {% if query.includeReferrals == "on" %}checked{% endif %}>
+      <label for="include-referrals">Include properties requiring an agency referral</label>
+      <span class="tooltip_entry">
+        <span class="icon_query"></span>
+        <span class="tooltip_content">
+          Some properties only accept applications by referral through an agency or case manager and do not allow the general public to apply.
+        </span>
+      </span>
+    </div>
+    <div>
+      <label for="property-name">Property Name</label>
+      <input type="text" aria-label="Property Name" id="property-name" value="{{query.propertyName | escape}}" name="propertyName">
+    </div>
+  </fieldset>
+
+  <div class="form_controls">
+    <input type="submit" id="filter-submit" class="btn btn_primary noprint" value="{{updateButtonStr}}"/>
+  </div>
+
+</form>
+
+
+
+
+  <div class="align-end flex stack-reversed" id="results">
```

Comment:
Make sure the element with id="results" is actually positioned above the results in order to provide some of the visual context of the search filters.  The jump to #results can be confusing and people may not realize they are on the same page and/or can scroll up still.  I think I placed the anchor within the form originally.  Feel free to rename the ID if you like.